java/linked-list/app/src/test/java/linked/list/AppTest.java

```diff
@@ -0,0 +1,253 @@
+/*
```

Comment:
Same as my other comment - the file name and the class name should be the same, so rename this file to "LinkedListTest.java"